x/stakeibc/keeper/update_redemption_rates_test.go

```diff
@@ -14,27 +15,28 @@ import (
 )
 
 type UpdateRedemptionRatesTestCase struct {
-	host_zone          stakeibc.HostZone
-	redemption_rate_t0 sdk.Dec
-	staked_bal         uint64
-	undelegated_bal    uint64
-	just_deposited_bal uint64
-	st_supply          uint64
+	hostZone         stakeibc.HostZone
+	redemptionRateT0 sdk.Dec
+	stakedBal        uint64
+	undelegatedBal   uint64
+	justDepositedBal uint64
+	stSupply         uint64
```

Comment:
Changes look good! Also you can remove these from this struct since they're defined in each function (and thus are already accessible as normal variables). 

Also, super nit, but you only have to return the hostzone id, not the whole object. 

```
type UpdateRedemptionRatesTestCase struct {
    hostZoneId  string
    allRecords  []recordtypes.DepositRecord
}
```

Reply:
good suggestion, done